src/kili/utils/labels/geojson/polygon.py

```diff
@@ -3,6 +3,35 @@
 from typing import Any, Dict, List, Optional
 
 
+def is_clockwise(vertices):
+    """Returns a boolean to indicate if the vertices are stored clockwise in the array.
+
+    This function uses the Shoelace formula, see also : https://en.wikipedia.org/wiki/Shoelace_formula
+    """
+    n = len(vertices)
+    sum_product = 0
+
+    for i in range(n):
+        x1, y1 = vertices[i]["x"], vertices[i]["y"]
+        x2, y2 = vertices[(i + 1) % n]["x"], vertices[(i + 1) % n]["y"]
+        sum_product += (x2 - x1) * (y2 + y1)
+
+    return sum_product > 0
+
+
+def order_counter_clockwise(vertices):
```

Comment:
this function assume that polygons are either drawn in clockwise or counter clockwise order.

The platform support drawing of polygons that are neither clock or counter clockwise ordered.
Exemple : 
<img width="761" alt="not-ordered-polygon" src="https://github.com/kili-technology/kili-python-sdk/assets/83211006/a9a9c02d-abec-41b0-b786-a3d68590f674">

As those polygons maybe not valid for geojson format, we should ignore them to avoid blocking export format, and add a warning to the user.